app/experimenter/nimbus-ui/src/components/PageHome/DirectoryTable/index.stories.tsx

```diff
@@ -10,27 +10,44 @@ import DirectoryTable, {
   DirectoryLiveTable,
 } from ".";
 import { mockDirectoryExperiments } from "../../../lib/mocks";
+import {
+  NimbusExperimentPublishStatus,
+  NimbusExperimentStatus,
+} from "../../../types/globalTypes";
 
 export default {
   title: "pages/Home/DirectoryTable",
   component: DirectoryTable,
   decorators: [withLinks],
 };
 
-export const Basic = () => (
-  <DirectoryTable experiments={mockDirectoryExperiments()} />
+const experiments = mockDirectoryExperiments();
+const draftExperiments = experiments.filter(
+  ({ status, publishStatus }) =>
+    status === NimbusExperimentStatus.DRAFT &&
+    publishStatus === NimbusExperimentPublishStatus.IDLE,
```

Comment:
When we finally get rid of `getStatus`, I wonder if we can/should set the pair of these (and the other status/publishStatus pairs) to something nice & readable for future state reference 🤔